bref/Dockerfile

```diff
@@ -1,17 +1,17 @@
 FROM bref/php-74-fpm-dev
 USER root
-RUN curl -SsL https://github.com/kvz/json2hcl/releases/download/v0.0.6/json2hcl_v0.0.6_linux_amd64 > /usr/local/bin/json2hcl \
-  && chmod 755 /usr/local/bin/json2hcl \
-  && json2hcl -version
-RUN curl -SsL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /usr/local/bin/jq \
-  && chmod 755 /usr/local/bin/jq \
-  && jq --version
+RUN curl -SsL https://github.com/kvz/json2hcl/releases/download/v0.0.6/json2hcl_v0.0.6_linux_amd64 > /usr/local/bin/json2hcl && \
+  chmod 755 /usr/local/bin/json2hcl && \
+  json2hcl -version
+RUN curl --compressed -SsL https://github.com/antonmedv/fx/releases/download/20.0.2/fx-linux.zip -o - | \
+  gunzip - > /usr/local/bin/fx && \
+  chmod 755 /usr/local/bin/fx
```

Comment:
I didn't ran fx --version the line after because of https://github.com/antonmedv/fx/pull/148